examples/text_to_image/train_text_to_image_lora_sdxl.py

```diff
@@ -706,6 +706,9 @@ def load_model_hook(models, input_dir):
     accelerator.register_save_state_pre_hook(save_model_hook)
     accelerator.register_load_state_pre_hook(load_model_hook)
 
+    if args.gradient_checkpointing:
+        unet.enable_gradient_checkpointing()
```

Comment:
The text encoder can be trainable in this script too:
https://github.com/huggingface/diffusers/blob/bf725e044e0b2fec29c074510c2d12d5ef66d2d1/examples/text_to_image/train_text_to_image_lora_sdxl.py#L621

This is how it's done:
https://github.com/huggingface/diffusers/blob/bf725e044e0b2fec29c074510c2d12d5ef66d2d1/examples/dreambooth/train_dreambooth_lora_sdxl.py#L976